### ISSUE TITLE
Defining an attribute without the additional Hash of options generates an unhelpful error

### ISSUE THREAD

**Issue by burtlo** [inspec/inspec#2738]
### Description

An `attribute` incorrectly defined in a control file does not provide details in the error message that the incorrect definition is the exact cause of the problem.

### InSpec and Platform Version

InSpec 2.0.17, Mac 0SX 10.13.13

### Replication Case

I define an attribute with a control file of my profile without the Hash of additional options.

```ruby
webserver_id = attribute('ec2_instance.webserver_id')

describe aws_ec2_instance(webserver_id) do
  it { should be_running }
end
```

When I execute this control with the profile an error message is generated.

```shell
$ inspec exec training-aws-security --attrs abc.yml
wrong number of arguments (given 1, expected 2)
```

### Possible Solutions

Describe what attribute is need of additional data with the control file and line number. Suggest a definition.

OR

Allow for an attribute to be defined without the additional Hash of data. In this instance I don't want to set a default value and the description may not be that important to this situation.

### Workarounds

```ruby
webserver_id = attribute('ec2_instance.webserver', {})
```

**Comment by clintoncwolfe** [comment]
Thanks for the report!   That would be a very opaque and frustrating experience for the user.  Should be an easy fix.

**Comment by kekaichinose** [comment]
This issue has aged. @burtlo Have you tried out using the new inputs system in Chef InSpec 4.0? In this case, the second option, which you're not wanting to set there, would be optional so I think it would address your use case. I don't think the resulting error text would be valid in this scenario anymore either since inputs can just take one argument.

Please let us know if this issue is resolved for you in Chef InSpec 4.0. If we do not hear back from you in the next 24 hours, we'll close the issue. Thanks!

**Comment by clintoncwolfe** [comment]
This issue is not present in inspec 4, as inputs are routinely accessed by name only.
